src/components/dialogs/generation-dispatch/generation-dispatch-form.js

```diff
@@ -4,48 +4,96 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-import TextInput from '../../utils/rhf-inputs/text-input';
-import { LOSS_COEFFICIENT } from '../../utils/field-constants';
+import { EQUIPMENT_TYPES } from '../../utils/equipment-types';
+import DirectoryItemsInput from '../../utils/rhf-inputs/directory-items-input';
+import FloatInput from '../../utils/rhf-inputs/float-input';
+import {
+    LOSS_COEFFICIENT,
+    DEFAULT_OUTAGE_RATE,
+    GENERATORS_WITHOUT_OUTAGE,
+} from '../../utils/field-constants';
 import {
     gridItem,
     percentageTextField,
-    standardTextField,
+    GridSection,
 } from '../../dialogs/dialogUtils';
 import Grid from '@mui/material/Grid';
 import {
     formatPercentageValue,
     isValidPercentage,
 } from '../percentage-area/percentage-area-utils';
+import { elementType } from '@gridsuite/commons-ui';
 import { useFormContext } from 'react-hook-form';
 import React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+
+export const useStyles = makeStyles((theme) => ({
+    padding: {
+        paddingTop: theme.spacing(2),
+    },
+}));
 
 const GenerationDispatchForm = ({ currentNode, studyUuid }) => {
+    const classes = useStyles();
     const { setValue } = useFormContext();
 
-    const handleLossCoefficientValueChange = (value) => {
-        const lossCoefficientValue = formatPercentageValue(value);
-        setValue(LOSS_COEFFICIENT, lossCoefficientValue, {
+    const handleCoefficientValueChange = (id, value) => {
+        const coefficientValue = formatPercentageValue(value);
+        setValue(id, coefficientValue, {
             shouldValidate: true,
         });
-        return lossCoefficientValue;
+        return coefficientValue;
     };
```

Comment:
why do we need setValue in outputTransform ?
it means we are calling "setValue" right before setting the value in the FloatInput
I don't think it's necessary

Reply:
You're right
I have removed this unneccessary code